app/lib/dns.server.ts

```diff
@@ -378,6 +381,11 @@ export const isNameValid = (fqdn: string, username: string) => {
   }
   const subdomain = fqdn.substring(0, fqdn.length - toRemove.length);
 
+  // Decline if subdomain has inappropriate word(s)
+  if (filter.isProfane(subdomain)) {
```

Comment:
To confirm, this won't check the username portion, right?  We don't want to fail something that is someone's name by accident.

This is only the `foo` in `foo.username.root.domain` right?

Reply:
Yes, only `subdomain` will be checked